js_modules/dagit/packages/core/src/partitions/PartitionStatus.tsx

```diff
@@ -13,9 +13,34 @@ type SelectionRange = {
 
 const MIN_SPAN_WIDTH = 8;
 
+export enum PartitionState {
```

Comment:
Previously, this component used RunStatus and did a mapping to colors. I added this intermediate enum so that run states and asset states can be coerced into the same type and passed to this component.  Right now the asset version only uses missing and success, but with the new work Sean is doing I think we'll have failing / "out of date" soon.